### PR TITLE
Canvas game

### PR DESCRIPTION
**Pull request by joewagner**
Merging canvas game into main so folks can see it more easily

### CODE REVIEW

**Comment by dtbuchholz** [comment]
@joewagner @andrewxhill are we good to merge this? i wanna throw the link into our docs as a reference but don't want to link to this "canvas" branch but to main

**Comment by joewagner** [comment]
@dtbuchholz Sorry this has been delayed.  Would you want to give this a thorough review?  If it looks good to you let's merge.

File: canvas-game/contracts/CanvasGame.sol [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+  PausableUpgradeable,
+  ReentrancyGuardUpgradeable,
+  UUPSUpgradeable
+{
+    using Counters for Counters.Counter;
+    Counters.Counter private _tokenIds;
+    ITablelandTables private _tableland;
+
+    string private _baseURIString;
+    string private _metadataTable;
```
Comment (andrewxhill) [inline canvas-game/contracts/CanvasGame.sol:28]: i don't see '_metadataTable' ever getting set

**Comment by sanderpick** [comment]
should this one be merged?

**Comment by dtbuchholz** [comment]
Oops, sorry, let this slip a bit. Merging.

**Comment by sanderpick** [comment]
nre, i saw a chess game on the holochain network telegram... might be interesting to take a look at https://github.com/eyss/elemental-chess

**Pull request merged**